### PR TITLE
Give Elixir changes some background

### PR DESCRIPTION
**Pull request by neenjaw**
@iHiD, PR as requested.  Is this the information, context that you were looking for?

### CODE REVIEW

File: using/solving-exercises/legacy-files.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ Deleting a file simply removes it from the **next** iteration that you submit vi
 
 ### Elixir
 
-At the launch of v3, Elixir changed its exercises.
+During the course of v2, Elixir changes its exercise structure from using elixir scripts (`*.exs` files) to a complete mix projects for each exercise.
```
Comment (ErikSchierboom) [inline using/solving-exercises/legacy-files.md]:
```suggestion
During the course of v2, Elixir changed its exercise structure from using elixir scripts (`*.exs` files) to a complete mix project for each exercise.
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Eh, @neenjaw @angelikatyborska you've both submitted PRs to add the same information (see https://github.com/exercism/docs/pull/219/files for Angelika's PR). Which one do you want me to merge?

**Comment by angelikatyborska** [comment]
Oops, I wasn't tagged in this PR so I didn't notice it. I would like to propose a merge of both texts. I like Tim's mention of not having to move any tests files, but I would want to also mention the specific month and year. Let me add a suggestion comment to this PR.

**Comment by iHiD** [comment]
Could you pull that together @angelikatyborska ? (I know you said you have a busy day at work so say if it's not possible)

**Comment by iHiD** [comment]
(And tim will be asleep for a few hours now)

File: using/solving-exercises/legacy-files.md [hunk 2]
```diff
@@ -39,7 +39,9 @@ Deleting a file simply removes it from the **next** iteration that you submit vi
...
+In order to use the features of v3, copy the your solution module to the stub file located at `lib/<exercise_stub>.ex`.
+You don't need to move the test file, the updated version will already be present in the correct location.
```
Comment (angelikatyborska) [inline using/solving-exercises/legacy-files.md]:
```suggestion
In July 2019, Elixir changed its exercise structure from using Elixir scripts (`*.exs` files) to a complete [Mix project](https://elixir-lang.org/getting-started/mix-otp/introduction-to-mix.html) for each exercise.
In order to use the features of v3, copy your solution to the stub file located at `lib/<exercise_stub>.ex`.
You don't need to move the test file, the updated version will already be present in the correct location.
```

Comment (angelikatyborska) [reply using/solving-exercises/legacy-files.md]: @iHiD take this suggestion and merge 🙂

**Comment by iHiD** [comment]
Thank you both!

**Pull request merged**